src/v/kafka/server/tests/list_offsets_test.cc

```diff
@@ -137,6 +137,46 @@ FIXTURE_TEST(list_offsets_latest, redpanda_thread_fixture) {
     BOOST_CHECK(resp.data.topics[0].partitions[0].offset > model::offset(0));
 }
 
+FIXTURE_TEST(list_offsets_not_found, redpanda_thread_fixture) {
+    wait_for_controller_leadership().get0();
```

Comment:
nit: `.get()` is preferable for new code (there's a few of these).

Reply:
Fixed